web/packages/selfhosted/test/polyfill/document_embeds/expected.html

```diff
@@ -0,0 +1,2 @@
+<li>There are 3 embeds for document.embeds</li>
```

Comment:
Can't we do a simple assertion instead of writing this to the document and comparing HTML later?